clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -56,10 +57,13 @@
                                                                           new ArrayOf(STRING),
                                                                           "An array of topics to fetch metadata for. If no topics are specified fetch metadata for all topics."));
 
-    public static final Schema BROKER = new Schema(new Field("node_id", INT32, "The broker id."),
+    public static final Schema METADATA_REQUEST_V1 = new Schema(new Field("topics",
+                                                                          new ArrayOf(STRING, true),
+                                                                          "An array of topics to fetch metadata for. If the topics array is null fetch metadata for all topics."));
+
+    public static final Schema METADATA_BROKER_V0 = new Schema(new Field("node_id", INT32, "The broker id."),
```

Comment:
The idea is that this is the broker used with the metadata response. All objects tied to the metadata response are prefixed with METADATA. There are other broker objects in this file that I would not want to be confused.